mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java

```diff
@@ -78,8 +80,10 @@
     @BindView(R.id.ll_center)
     LinearLayout llCenter;
 
-    @BindView(R.id.layout_submission)
-    LinearLayout layout_submission;
+//    @BindView(R.id.layout_submission)
```

Comment:
Never leave comments like these. Comments should be something meaningful and not just unused code because, over time, this clutter will become difficult to handle.

Reply:
Ok